vgscollect/src/main/java/com/verygoodsecurity/vgscollect/core/model/VGSDate.kt

```diff
@@ -0,0 +1,72 @@
+package com.verygoodsecurity.vgscollect.core.model
+
+import java.util.*
+
+public class VGSDate private constructor(
+    val day: Int,
+    val month: Int,
+    val year: Int
+) : Comparable<VGSDate> {
+
+    companion object {
+
+        fun createDate(day: Int, month: Int, year: Int): VGSDate? {
```

Comment:
@DonaldRG I think `Date` in this function name is redundant, as this function is inside `VGSDate` class. So I think just `create` will be enough.